invenio.cfg

```diff
@@ -445,3 +445,8 @@ RDM_RECORDS_IDENTIFIERS_SCHEMES = {**RDM_RECORDS_IDENTIFIERS_SCHEMES,
                                       "inspire": {"label": _("Inspire"),
                                                   "validator": schemes.is_inspire,
                                                   "datacite": "INSPIRE"}}}
+
+RDM_RECORDS_PERSONORG_SCHEMES = {**RDM_RECORDS_PERSONORG_SCHEMES,
+                                 **{"inspire": {"label": _("Inspire"),
+                                                "validator": schemes.is_inspire,
```

Comment:
I doubt that there is any detection in place?

Reply:
this is validation, not detection. Basically it checks if the ID is a number, otherwise it fails